code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm

```diff
@@ -0,0 +1,305 @@
+					to_chat(elitemind, "<b>While the opponent might have an upper hand with  powerful mining equipment and tools, you have great power normally limited by AI mobs.</b>")
+					to_chat(elitemind, "<b>If you want to win, you'll have to use your powers in creative ways to ensure the kill.  It's suggested you try using them all as soon as possible.</b>")
+					to_chat(elitemind, "<b>Should you win, you'll receive extra information regarding what to do after.  Good luck!</b>")
+					sleep(100)
+				else
+					visible_message("<span class='boldwarning'>The stirring stops, and nothing emerges.  Perhaps try again later.</span>")
+					activity = 0
+					activator = null
+					return
+			else
+				sleep(30)
+			var/selectedspawn = pick(potentialspawns)
+			mychild = new selectedspawn(loc, src)
+			mychild.myparent = src
+			visible_message("<span class='boldwarning'>[mychild] emerges from the crevice!</span>")
+			playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, 1, 1)
+			if(boosted == 1)
+				mychild.key = elitemind.key
+				mychild.sentience_act()
+			INVOKE_ASYNC(src, .proc/arena_trap)
+			return
+	if(activity == 2)
+		if(istype(user, /mob/living/carbon/human))
+			activity = 1
+			visible_message("<span class='boldwarning'>The crevice glows as your arm enters its radius.  Your instincts tell you to step back.</span>")
+			activator = user
+			INVOKE_ASYNC(src, .proc/arena_trap)
+			if(boosted == 1)
+				SEND_SOUND(mychild, sound('sound/effects/magic.ogg'))
+				to_chat(mychild, "<b>Someone has activated your crevice.  You will be returned to fight shortly, get ready!</b>")
+				sleep(40)
+			sleep(30)
+			mychild.forceMove(loc)
+			visible_message("<span class='boldwarning'>[mychild] emerges from the crevice!</span>")
+			playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, 1, 1)
+			mychild.revive(full_heal = 1, admin_revive = 1)
+			if(boosted == 1)
+				mychild.maxHealth = mychild.maxHealth * 2
+				mychild.health = mychild.maxHealth
+			
+			return
+			
+/obj/structure/elite_crevice/Initialize()
+	. = ..()
+	AddComponent(/datum/component/gps, "Menacing Signal")
+		
+/obj/structure/elite_crevice/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/organ/regenerative_core) && activity == 0 && boosted == 0)
+		var/obj/item/organ/regenerative_core/core = W
+		if(core.preserved == 1)
+			visible_message("<span class='boldwarning'>As [user] drops the core into the crevice, the red light intensifies for a brief moment, then returns to normal.</span>")
+			boosted = 1
+			qdel(core)
+			
+/obj/structure/elite_crevice/proc/arena_trap()
+	var/turf/T = get_turf(src)
+	if(T && activity == 1)
+		for(var/t in RANGE_TURFS(12, T))
+			if(t && get_dist(t, T) == 12)
+				var/obj/effect/temp_visual/elite_crevice_wall/newwall
+				newwall = new /obj/effect/temp_visual/elite_crevice_wall(t, src)
+				newwall.activator = src.activator
+				newwall.ourelite = src.mychild
+		sleep(100)
```

Comment:
timers

Reply:
Done for upcoming fix.

Reply:
You should know that you don't have to comment on everything like this. When you change the file in a commit, git should automatically mark addressed comments as resolved (or you can do it yourself manually). Commenting can spam notifications

Reply:
Haha, thanks for the heads up.